lib/Doctrine/Migrations/Stopwatch.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations;
+
+use Symfony\Component\Stopwatch\Stopwatch as SymfonyStopwatch;
+use Symfony\Component\Stopwatch\StopwatchEvent;
+
+class Stopwatch
+{
+    public function start(string $section) : StopwatchEvent
+    {
+        $stopwatch = new SymfonyStopwatch(true);
```

Comment:
why creating a new SymfonyStopwatch each time ?

Reply:
@stof This may have been a misunderstanding on my part of how Stopwatch behaves. Looking at the code now I see that I can have one instance of it. I will make a PR to change it.